website/docs/r/cloud_project_containerregistry_oidc.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+subcategory : "Managed Private Registry"
+---
+
+# ovh_cloud_project_containerregistry_oidc
+
+Creates an OIDC configuration in an OVHcloud Managed Private Registry.
+
+## Example Usage
+
+```hcl
+resource "ovh_cloud_project_containerregistry_oidc" "my-oidc" {
+  service_name = "XXXXXX"
+  registry_id  = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx"
+
+  #required field
+  oidc_name          = "my-oidc-provider"
+  oidc_endpoint      = "https://xxxx.yyy.com"
```

Comment:
```suggestion
  oidc_endpoint      = "https://xxxx.yyy.com/realms/myrealm"
```

Reply:
I didn't add it because it seems to be required by Keycloak but I'm not sure it should be added for others OIDC providers